johnwayodi/store-manager
# #161078989 Add Product Details view for Store Attendant

#### What does this PR do?
Adds a Product Detials Page for the Store Attendant where the Store Attendant can also choose to add the product to Shopping cart 

#### Description of Task to be completed?
_**Back to Dashboard**_  should take attendant back to his dashboard
_**Back to Products**_  should take attendant back to the products view
_**Add to Cart**_  should add the product and the specified quantity to cart

#### How should this be manually tested?
1. `git clone https://github.com/johnwayodi/store-manager.git`
2. Ensure you are in the directory where you cloned this repository, _**store-manager**_ by default
2. `git checkout ft-product-details-161078989`
3. open product-detail.html on browser
#### Any background context you want to provide?
The Product Details page is accessible only to the Store Attendant from Dashboard

#### What are the relevant pivotal tracker stories?
#161078989
#### Screenshots (if appropriate)
![product-details](https://user-images.githubusercontent.com/10683903/46744573-11d2b000-ccb4-11e8-9f45-1062a0dc6ecf.png)

#### Questions:
Any feedback on what I change? Drop a comment or suggestion below
